.travis.yml

```diff
@@ -72,7 +72,7 @@ before_install:
   # PrestaShop configuration
   - cp tests-legacy/parameters.yml.travis app/config/parameters.yml
 
-before_script:
+install:
```

Comment:
Why have you changed that ?

Reply:
Because dependencies must be installed in travis' `install` step

Reply:
:+1: